net-p2p/bitcoind/bitcoind-0.21.0.ebuild

```diff
@@ -0,0 +1,171 @@
+"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc ~ppc64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="+asm examples knots +sqlite system-leveldb test upnp +wallet zeromq"
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	acct-group/bitcoin
+	acct-user/bitcoin
+	>=dev-libs/boost-1.58.0:=[threads(+)]
+	dev-libs/libevent:=
+	>dev-libs/libsecp256k1-0.1_pre20200911:=[recovery,schnorr]
+	>=dev-libs/univalue-1.0.4:=
+	system-leveldb? ( virtual/bitcoin-leveldb )
+	sqlite? ( >=dev-db/sqlite-3.7.17:= )
+	upnp? ( >=net-libs/miniupnpc-1.9.20150916:= )
+	wallet? ( sys-libs/db:$(db_ver_to_slot "${DB_VER}")=[cxx] )
+	zeromq? ( net-libs/zeromq:= )
+"
+RDEPEND="${DEPEND}"
+BDEPEND="
+	>=sys-devel/autoconf-2.69
+	>=sys-devel/automake-1.13
+"
+
+DOCS=(
+	doc/bips.md
+	doc/bitcoin-conf.md
+	doc/descriptors.md
+	doc/files.md
+	doc/JSON-RPC-interface.md
+	doc/psbt.md
+	doc/reduce-traffic.md
+	doc/release-notes.md
+	doc/REST-interface.md
+	doc/tor.md
+)
+
+S="${WORKDIR}/bitcoin-${BITCOINCORE_COMMITHASH}"
+
+pkg_pretend() {
+	if use knots; then
+		elog "You are building ${PN} from Bitcoin Knots."
+		elog "For more information, see:"
+		elog "https://bitcoinknots.org/files/0.21.x/${KNOTS_PV}/${KNOTS_P}.desc.html"
+	else
+		elog "You are building ${PN} from Bitcoin Core."
+		elog "For more information, see:"
+		elog "https://bitcoincore.org/en/2021/01/14/release-${PV}/"
+	fi
+	elog "Replace By Fee policy is now always enabled by default: Your node will"
+	elog "preferentially mine and relay transactions paying the highest fee, regardless"
+	elog "of receive order. To disable RBF, set mempoolreplacement=never in bitcoin.conf"
+}
+
+src_prepare() {
+	sed -i 's/^\(complete -F _bitcoind bitcoind\) bitcoin-qt$/\1/' contrib/${PN}.bash-completion || die
+
+	local knots_patchdir="${WORKDIR}/${KNOTS_P}.patches/"
+
+	eapply "${knots_patchdir}/${KNOTS_P}.syslibs.patch"
+
+	if use knots; then
+		eapply "${knots_patchdir}/${KNOTS_P}.f.patch"
+		eapply "${knots_patchdir}/${KNOTS_P}.branding.patch"
+		eapply "${knots_patchdir}/${KNOTS_P}.ts.patch"
+	fi
+
+	default
+
+	echo '#!/bin/true' >share/genbuild.sh || die
+	mkdir -p src/obj || die
+	echo "#define BUILD_SUFFIX gentoo${PVR#${PV}}" >src/obj/build.h || die
+
+	eautoreconf
+	rm -r src/secp256k1 || die
+	if use system-leveldb; then
+		rm -r src/leveldb || die
+	fi
+}
+
+src_configure() {
+	local my_econf=(
+		$(use_enable asm)
+		--without-qtdbus
+		--without-qrencode
+		$(use_with upnp miniupnpc)
+		$(use_enable upnp upnp-default)
+		$(use_enable test tests)
+		$(use_enable wallet)
+		$(use_enable zeromq zmq)
+		--with-daemon
+		--disable-util-cli
+		--disable-util-tx
+		--disable-util-wallet
+		--disable-bench
+		--without-libs
+		--without-gui
+		--disable-fuzz
+		--disable-ccache
+		--disable-static
+		$(use_with sqlite)
+		$(use_with system-leveldb)
+		--with-system-libsecp256k1
+		--with-system-univalue
+	)
+	econf "${my_econf[@]}"
+}
+
+src_install() {
+	default
+
+	rm -f "${ED}/usr/bin/test_bitcoin" || die
```

Comment:
Same here. Is it somehow predictable when this file will exist and be installed? Via test phase?